### PR TITLE
Development

### PR DESCRIPTION
**Pull request by GregorySDTaylor**
Self-assessment: 2 (The assignment meets the standard and shows an understanding of media queries and break points)

The site looks OK in a browser/phone, but the CSS and HTML selectors are a mess.  I can see much more clearly now the problem that BEM tries to solve.  My biggest question right now is how to name these classes: using BEM, would I be identifying the actual item (e.g. workhistory in my resume) more or the content structure on the page (e.g. contentblock)?  It seems like the content structure approach would be more useful for single CSS files across multiple HTML sites, but that the actual item approach would be easier to read.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,348 @@
+* {
+    margin: 0;
+    Padding: 0;
+    font-family: "Open Sans", helvetica, arial, sans-serif;
+    font-weight: lighter;
+}
+
+Img {
```
Comment (amygori) [inline styles.css:8]: You've got a typo here with the capitalization of `img` -- this style isn't being applied.

**Review by amygori — APPROVED** [review]
Your code (and the design!) look good. You did a nice job of making your code clean and easy to read, and your thoughtfulness with class naming is something that definitely pays off. 👏

**Comment by amygori** [comment]
@lifeprolific Yeah, class names require a lot of thought -- and if you give them a lot of thought it will make you a better developer! BEM is a good system, though you may eventually find one you like better. In that approach, a block is a modular, reusable section -- for instance, a footer or a navigation menu. if it's something that you could drop in, as is, to page and is not dependent on its context, then it's probably a block, and its name should describe its purpose. An element is a component of the block, dependent on its context, and then a modifier is like a flavor or variation or either an element or a block (or both). Both elements and modifiers are optional parts of the class name. The other thing that might be useful to know is that having more classnames is better. Then you can apply, or not apply, classes as needed and in different combinations (again, thinking more modularly) to any element. Here are a couple articles on BEM -- maybe you've seen them already, but they are ones I like a lot: 

- [CSS-Tricks BEM 101](https://css-tricks.com/bem-101)
- [Intro to BEM Methodology](https://webdesign.tutsplus.com/articles/an-introduction-to-the-bem-methodology--cms-19403)

**Comment by GregorySDTaylor** [comment]
@amygori Thank you so much for your thoughtful comments!

**Pull request merged**